ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/results-export/container.tsx

```diff
@@ -137,32 +137,35 @@ class ResultsExport extends React.Component<Props, State> {
       },
     ]
 
+    const attributes = Array.from(
+      new Set(
+        this.props.results
+          .map(result => result.attributes)
+          .reduce((result, arr) => result.concat(arr))
+      )
+    )
+    let argBody: any
+    let transformer: string
     if (this.props.isZipped) {
-      response = await exportResultSet('zipCompression', {
-        searches,
-        count,
-        args: {
-          transformerId: uriEncodedTransformerId,
-        },
-      })
+      transformer = 'zipCompression'
+      argBody = {
+        columnOrder: attributes,
+        columnAliasMap: properties.attributeAliases,
+        transformerId: transformerId,
+      }
     } else {
-      const attributes = Array.from(
-        new Set(
-          this.props.results
-            .map(result => result.attributes)
-            .reduce((result, arr) => result.concat(arr))
-        )
-      )
-      response = await exportResultSet(uriEncodedTransformerId, {
-        searches,
-        count,
-        sorts: [{ attribute: 'modified', direction: 'descending' }],
-        args: {
-          columnOrder: attributes,
-          columnAliasMap: properties.attributeAliases,
-        },
-      })
+      transformer = transformerId
+      argBody = {
+        columnOrder: attributes,
+        columnAliasMap: properties.attributeAliases,
+      }
     }
+    response = await exportResultSet(transformer, {
```

Comment:
❓ Since this is an I/O operation would it make sense to handle an exception here, i.e. `try/catch`?

Reply:
agreed, there should be some exception handling here - i think the plan is to merge this pr and @abel-connexta is going to address some additional issues (like when this operation fails when we don't have thumbnails/location data and try to export with formats that use them)